Hyperlint Automation: Meta Description Fixes (#20218)
cloudflare/cloudflare-docs: src/content/docs/ruleset-engine/rules-language/actions.mdx

head:
  - tag: title
    content: Actions reference

> description: Learn about Cloudflare Ruleset Engine actions, including Block, Skip, and Log. Understand how actions handle rule matches and configure security features.

Reviewer: ```suggestion
description: Learn about actions supported by the Rules language, including Block, Skip, and Log.
```
